cfn-resources/cfn-submit-helper.sh

```diff
@@ -15,15 +15,17 @@
 # Example with DEBUG logging enabled by default for set of resources:
 # LOG_LEVEL=debug ./cfn-submit-helper.sh project database-user project-ip-access-list cluster network-peering
 #
+set -xeo
 
+# shellcheck source=/dev/null
 . ./cfn-submit-helper.config
 env | grep CFN_SUBMIT_
 #echo "AWS_DEFAULT_PROFILE=${AWS_DEFAULT_PROFILE}"
 
 _DRY_RUN=${DRY_RUN:-false}
 _BUILD_ONLY=${BUILD_ONLY:-false}
 _SUBMIT_ONLY=${SUBMIT_ONLY:-false}
-_CLOUD_PUBLISH=${CLOUD_PUBLISH:-false}
+#_CLOUD_PUBLISH=${CLOUD_PUBLISH:-false}
```

Comment:
why commenting this?

Reply:
shellscript was complaining about `_CLOUD_PUBLISH` not being used. I am not 100% sure why is there and I want to wait to discuss all these scripts with consultants before refactoring them

Reply:
it probably makes sens then to delete if unused

Reply:
updated